AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/Consumer/CloudWatchLoggingConsumer.swift

```diff
@@ -22,11 +22,11 @@ class CloudWatchLoggingConsumer {
         client: CloudWatchLogsClientProtocol,
         logGroupName: String,
         userIdentifier: String?
-    ) throws {
+    ) async throws {
```

Comment:
Just wondering, does this need to `throw`? I don't see any `try` in the init 🤔

Reply:
good catch, removed the `throws` and `try`